package/yast2-schema.spec

```
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-schema
-Version:        4.1.7
+Version:        4.1.8
```

Um, when we change the dependency to something else than in SP1 we should also bump this package version. =
```suggestion
Version:        4.2.0
```